src/lamp_py/tableau/hyper.py

```diff
@@ -276,6 +236,91 @@ def run_hyper(self) -> None:
                 if retry_count == max_retries:
                     process_log.log_failure(exception=exception)
 
+
+class ParquetHyperJob(HyperJob):  # pylint: disable=R0902
+    """
+    Publish a remote Parquet file to Tableau as a HyperFile
+    """
+
+    def __init__(
+        self,
+        hyper_file_name: str,
+        remote_parquet_path: str,
+        lamp_version: str,
+    ) -> None:
+        HyperJob.__init__(
+            self,
+            hyper_file_name=hyper_file_name,
+            remote_parquet_path=remote_parquet_path,
+            lamp_version=lamp_version,
+        )
+
+    def parquet_schema(self) -> pyarrow.schema:
+        active_fs = fs.S3FileSystem()
+        dataset = pq.ParquetDataset(
+            self.remote_parquet_path, filesystem=active_fs
+        )
+        return dataset.schema
+
+
+class RdsHyperJob(HyperJob):  # pylint: disable=R0902
```

Comment:
New base class for HyperJobs relying on the Performance Manager RDS

Reply:
I'm not sure why we need this separate `RdsHyperJob` class. The previous implementation didn't have any details in the `HyperJob` class that were specific to using an RDS. All of the RDS specific details were contained in either the `create_parquet` or `update_parquet` abstract methods.

Reply:
i don't think either of those methods are needed by the job, since the parquet file has already been created and uploaded to s3. i could leave them as unimpled, but that feels wrong. i think with the `ParquetHyperJob` we could hook it up to other parquet files generated by other parts of our pipeline without having to create new subclasses. we just have to point it at a remote file and provide version and tableau hyperfile names.

we also don't have to define the parquet schema for a second time, since we can pull it directly from the file.

Reply:
Essentially the whole `process_alerts` function of the `alerts.py` file is re-implementing the `create_parquet` and `update_parquet` functionality of the `HyperJob` base class, just in a different location.